angular/src/app/components/asset-detail/asset-detail.component.ts

```diff
@@ -35,6 +35,7 @@ export class AssetDetailComponent implements OnInit {
       this.feature = next;
       try {
         let featureSource = this.envService.apiUrl + '/assets/' + this.feature.assets[0].path;
+
```

Comment:
Is this blank line intentional mid-block?

Reply:
dropping 👍